src/index.html

```diff
@@ -1,18 +1,68 @@
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="html"
+>
   <head>
     <meta charset="UTF-8" />
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100..900&display=swap"
+      rel="stylesheet"
+    />
     <title>Product cards</title>
+
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="../src/styles/styles.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <conteiner class="container">
+      <div class="image"></div>
+      <div class="page">
+        <div class="name">
+          <p class="name__apple">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </p>
+        </div>
+        <div
+          class="kod"
```

Comment:
Different class, use more descriptive name for classes everywhere
```suggestion
          class="card__code"
```